src/editor/Store/CommandStoreConnector.re

```diff
@@ -60,6 +60,23 @@ let start = _ => {
     });
   };
 
+  let windowMoveEffect = (state: State.t, direction, _) => {
+    Isolinear.Effect.createWithDispatch(~name="window.move", dispatch => {
```

Comment:
If we want to get picky, I could stringify the `direction` and append that to the effect name.

Reply:
Cool idea, thanks for thinking about it @KevinSjoberg !

The main idea with the `name` is to have it show up in our logs when we run effects (logging when an effect begins / ends and how long it takes, etc). If it ends up that we need more granular logging - we could add the direction as you suggest to the name, or log the direction in the effect - but I'm OK leaving as-is for now until we need it 👍